lib/bulkhead/BulkheadOptions.ts

```diff
@@ -24,11 +24,6 @@ export type BulkheadOptions = {
   onError?: 'throw' | 'reject' | 'ignore' | 'ignoreAsync',
 };
 
-/**
- * Limits the number of queued concurrent executions of a method.
- * When the limit is reached the execution is delayed and queued.
- * @param threshold the max number of concurrent executions.
- */
-export function bulkhead(threshold: number) {
-  throw new Error('Not implemented.');
-}
+export const DEFAULT_SIZE = 0;
+export const DEFAULT_SCOPE = 'instance';
```

Comment:
The documentation above at line 4 says `class` is default, but here `instance` is set.